src/android.ts

```diff
@@ -310,10 +310,21 @@ export const androidApi = <RunTarget extends SupportedRunTarget<'android'>>(
             // Install app if necessary.
             if (!(await this.isAppInstalled('com.wireguard.android'))) {
                 try {
-                    const fdroidMeta = await fetch('https://f-droid.org/api/v1/packages/com.wireguard.android').then(
-                        (res) => res.json()
-                    );
-                    const apkUrl = `https://f-droid.org/repo/com.wireguard.android_${fdroidMeta.suggestedVersionCode}.apk`;
+                    const fdroidVersionCode = await fetch('https://f-droid.org/api/v1/packages/com.wireguard.android')
+                        .then((res) => res.json())
+                        .then((fdroidMeta) => {
+                            if (
+                                fdroidMeta.packages.some(
+                                    (p: { versionCode: number }) => p.versionCode === fdroidMeta.suggestedVersionCode
+                                )
+                            )
+                                return fdroidMeta.suggestedVersionCode;
+
+                            return fdroidMeta.packages.sort((a: { versionCode: number }, b: { versionCode: number }) =>
+                                a.versionCode === b.versionCode ? 0 : a.versionCode > b.versionCode ? 1 : -1
+                            )[0].versionCode;
+                        });
```

Comment:
```suggestion
                            return fdroidMeta.packages.sort(
                                (a: { versionCode: number }, b: { versionCode: number }) =>
                                    b.versionCode - a.versionCode
                            )[0].versionCode;

```

Reply:
This is also the wrong way around, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

Reply:
https://pastebin.com/raw/gJGddmEF